src/core/thermostat.cpp

```diff
@@ -209,8 +247,17 @@ void thermo_heat_up()
 {
   int j;
   if(thermo_switch & THERMO_LANGEVIN) {
+#ifndef PARTICLE_ANISOTROPY
     langevin_pref2_buffer          = langevin_pref2;
     langevin_pref2 *= sqrt(3);
```

Comment:
I believe it's related to the variance of the random number generator. How and why I don't know.